src/app/shared/clarin-item-box-view/clarin-item-box-view.component.ts

```diff
@@ -200,11 +200,11 @@ export class ClarinItemBoxViewComponent implements OnInit {
         getFirstCompletedRemoteData(),
         switchMap((clList: RemoteData<PaginatedList<ClarinLicense>>) => clList?.payload?.page))
       .subscribe(clarinLicense => {
+        let iconsList = [];
         clarinLicense.extendedClarinLicenseLabels.forEach(extendedCll => {
-          this.licenseLabelIcons.push(extendedCll?.icon);
+          iconsList.push(extendedCll?.icon);
```

Comment:
why `?` here?

Reply:
That's the check if the object is not null or undefined